test/e2e/puppeteer-error-snapshot/actor/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:20 AS builder
```

Comment:
this should use the beta image

Reply:
Do you mean `apify/actor-node-puppeteer-chrome:20-beta`?

Reply:
oh damn i am blind, i though its our image, lets keep it as is 👍